src/Compilers/CSharp/Portable/PublicAPI.Unshipped.txt

```diff
@@ -5,8 +5,8 @@ Microsoft.CodeAnalysis.CSharp.CSharpCompilationOptions.CSharpCompilationOptions(
 Microsoft.CodeAnalysis.CSharp.CSharpCompilationOptions.NullableContextOptions.get -> Microsoft.CodeAnalysis.CSharp.NullableContextOptions
 Microsoft.CodeAnalysis.CSharp.CSharpCompilationOptions.WithNullableContextOptions(Microsoft.CodeAnalysis.CSharp.NullableContextOptions options) -> Microsoft.CodeAnalysis.CSharp.CSharpCompilationOptions
 Microsoft.CodeAnalysis.CSharp.LanguageVersion.CSharp8 = 800 -> Microsoft.CodeAnalysis.CSharp.LanguageVersion
-Microsoft.CodeAnalysis.CSharp.LanguageVersion.Preview = 2147483646 -> Microsoft.CodeAnalysis.CSharp.LanguageVersion
 Microsoft.CodeAnalysis.CSharp.LanguageVersion.LatestMajor = 2147483645 -> Microsoft.CodeAnalysis.CSharp.LanguageVersion
+Microsoft.CodeAnalysis.CSharp.LanguageVersion.Preview = 2147483646 -> Microsoft.CodeAnalysis.CSharp.LanguageVersion
```

Comment:
Are there meaningful changes in this file? Please revert changes that simply move lines around

Reply:
Every change to the PublicAPI files is supposed to be followed by a call to `sort-unshipped.cmd`. That's all I did here.

Reply:
Would you prefer I change that in a separate PR? Or as separate commit to make reviewing easier?

Reply:
> Every change to the PublicAPI files is supposed to be followed by a call to sort-unshipped.cmd. 

This is the first time I hear about that. I see very little benefit and a lot of negatives in doing this. People should be able to examine the diff and quickly see what has changed, not just in PRs, but in history as well. Merge tools should be able to merge non-conflicting API changes automatically.

>Would you prefer I change that in a separate PR? Or as separate commit to make reviewing easier?

I prefer that sorting was not done in this PR, or any other PR.

---
In reply to: [267972689](https://github.com/dotnet/roslyn/pull/34306#discussion_r267972689) [](ancestors = 267972689)

Reply:
Sorting is extremely helpful for resolving conflicts in merge PRs, as changes to the public API are extremely likely to be added in the same section without sorting.

Reply:
> Sorting is extremely helpful for resolving conflicts in merge PRs, as changes to the public API are extremely likely to be added in the same section without sorting.

I have an opposite experience. Also, when one merges additions from different branches he should be able to take both sides and move on, without having to run some other tools to sort, etc. 

---
In reply to: [268290574](https://github.com/dotnet/roslyn/pull/34306#discussion_r268290574) [](ancestors = 268290574)